src/physics/em/detail/MollerBhabha.hh

```diff
@@ -3,33 +3,55 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file MollerBhabhaInteractorPointers.hh
+//! \file MollerBhabha.hh
 //---------------------------------------------------------------------------//
 #pragma once
 
 #include "base/Macros.hh"
 #include "base/Types.hh"
+#include "physics/base/Types.hh"
+#include "physics/base/Units.hh"
 
 namespace celeritas
 {
+struct ModelInteractPointers;
+
+namespace detail
+{
 //---------------------------------------------------------------------------//
 /*!
  * Device data for creating an interactor.
  */
-struct MollerBhabhaInteractorPointers
+struct MollerBhabhaPointers
 {
+    //! Model ID
+    ModelId model_id;
+
     //! ID of an electron
     ParticleId electron_id;
-    //! ID of a gamma
-    ParticleId gamma_id;
-    // XXX additional data
+    //! ID of a positron
+    ParticleId positron_id;
+    // Electron mass * c^2 [MeV] 
+    real_type electron_mass_c_sq;
+    // Mininum energy limit
+    units::MevEnergy min_valid_energy_;
```

Comment:
For a struct, where data members are public, they shouldn't be suffixed with `_`.